packages/prism-editor/src/utils/utils.ts

```diff
@@ -0,0 +1,38 @@
+/*
+
+ MIT License
+
+ Copyright (c) 2021 Looker Data Sciences, Inc.
+
+ Permission is hereby granted, free of charge, to any person obtaining a copy
+ of this software and associated documentation files (the "Software"), to deal
+ in the Software without restriction, including without limitation the rights
+ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ copies of the Software, and to permit persons to whom the Software is
+ furnished to do so, subject to the following conditions:
+
+ The above copyright notice and this permission notice shall be included in all
+ copies or substantial portions of the Software.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ SOFTWARE.
+
+ */
+import { Language } from 'prism-react-renderer'
+
+export const getPrismLanguage = (language: string): Language => {
+  language = language.toLowerCase()
+  const unstyled = ['kotlin', 'csharp', 'swift']
+  // TODO revert back to `go` in generator language definitions instead of using this
+  if (language === 'golang') {
+    return 'go'
+  } else if (unstyled.includes(language)) {
+    return 'clike'
+  }
+  return language as Language
+}
```

Comment:
Hmm, this utility assumes that outside of a select few languages (Kotlin, csharp, swift and go), the Prism package will recognize all our language labels which might not always be the case. Let's add a test that ensures we are getting the desired language for all our codegenerator languages and fails if it encounters another Golang-like scenario